spec/presenters/navigation_presenter_spec.rb

```diff
@@ -11,6 +11,7 @@
         let(:user) { build(:user, personal_key: 'abcdef') }
 
         it 'has a link to reset personal key' do
+          create(:profile, :active, :verified, user: user)
           nav_item = navigation.navigation_items.first.children.last
           expect(nav_item.title).to eq(I18n.t('account.navigation.reset_personal_key'))
         end
```

Comment:
should we add a negative case (ex a password reset profile does not see this?)

Reply:
yep, that's a really good idea, will add

Reply:
Added in https://github.com/18F/identity-idp/pull/6452/commits/b72588d47cadd24fd0aa7aad6315d52c2237a075